packages/ios_platform_images/example/ios/RunnerTests/UIImage+ios_platform_imagesTests.m

```diff
@@ -1,43 +0,0 @@
-// Copyright 2013 The Flutter Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-
-@import ios_platform_images;
-@import XCTest;
-
-// The tests test the UIImage extension which is a public API intended for use from native code
-// outside of the plugin (see package README). Any change that requires changing existing tests
-// in this file (unless it's just to reflect changes to the test assets) is a BREAKING CHANGE for
-// the package.
```

Comment:
We may want to keep this ObjC test untouched, since it's testing public API and we want to make sure no accidental breaking changes in ObjC.

Reply:
Yes, this needs to stay; see https://github.com/flutter/packages/pull/4847#discussion_r1325857344

This conversion is masking the fact that this PR is breaking the public API in the way described in that comment.

Reply:
I returned the ObjC test and removed the test with Swift. If you find it necessary, I will also add the test with Swift.

Reply:
Just objc test should be fine as they are testing the same thing. Adding a swift version can cause maintenance overhead of keeping 2 files in sync.